### PR TITLE
Added a recipe

### PR DESCRIPTION
**Pull request by eprendes**
Added recipe and image for Café Con Leche

### CODE REVIEW

File: src/recipes/cafeconleche.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        {
+            "quantity": "1",
+            "measure": "cup",
+            "ingredient": "hot or steamed milk"
+        },
+        {
+            "quantity": "1-3",
+            "measure": "teaspoons",
+            "ingredient": "sugar, optional and to taste"
+        },
```
Comment (alfg) [inline src/recipes/cafeconleche.json]:
```suggestion
        }
```

File: src/recipes/cafeconleche.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    ],
+    "directions": [
+        "Prepare espresso shot, Cubans commonly use a Moka stovetop pot.",
+        "Add your sugar to bottom of drink cup. Heat milk, either by slowly warming on the stove, microwaving, or steaming.",
+        "Pour piping hot espresso over sugar and stir until dissolved. Then pour hot milk into the cup and combine."
+    ],
+    "image": "<cafeconleche.jpg>",
+    "keywords": [
+        "coffee",
+        "breakfast",
```
Comment (alfg) [inline src/recipes/cafeconleche.json]:
```suggestion
        "breakfast"
```

File: src/recipes/cafeconleche.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+            "measure": "teaspoons",
+            "ingredient": "sugar, optional and to taste"
+        },
+    ],
+    "directions": [
+        "Prepare espresso shot, Cubans commonly use a Moka stovetop pot.",
+        "Add your sugar to bottom of drink cup. Heat milk, either by slowly warming on the stove, microwaving, or steaming.",
+        "Pour piping hot espresso over sugar and stir until dissolved. Then pour hot milk into the cup and combine."
+    ],
+    "image": "<cafeconleche.jpg>",
```
Comment (alfg) [inline src/recipes/cafeconleche.json]:
```suggestion
    "image": "cafeconleche.jpg",
```

File: src/recipes/cafeconleche.json (hunk 1 above)
Comment (eprendes) [reply src/recipes/cafeconleche.json]: Thank you! Sorry, this was my first ever json file!

**Comment by alfg** [comment]
No problem. Thanks for the fixes and for contributing! :)

**Pull request merged**